spec/views/todos/index_html.erb_spec.rb

```diff
@@ -2,12 +2,8 @@
 
 RSpec.describe "todos/index" do
   before do
-    controller.singleton_class.class_eval do
```

Comment:
What this change has to do with dual booting?

Reply:
@bronzdoc because this does not work with Rails 6.2. I did some research and it is an old idiom recommended by an old version of rspec-rails. It is no longer recommended and it hasn't been recommended in a while. 

It was changed 9+ years ago. https://github.com/rspec/rspec-rails/pull/420